tools/syz-trace2syz/trace2syz.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2018 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+// syz-trace2syz converts strace traces to syzkaller programs.
+//
+// Simple usage:
+//	strace -o trace -s 65500 -v -xx -f ./a.out
```

Comment:
`-a 1` should reduce trace size a bit
not that it's super radical reduction, but since they are huge any reduction will be useful